cypress/integration/finance/funds/add-expense-class-to-budget.spec.js

```diff
@@ -0,0 +1,66 @@
+import NewFund from '../../../support/fragments/finance/funds/newFund';
+import Funds from '../../../support/fragments/finance/funds/funds';
+import DateTools from '../../../support/utils/dateTools';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import testType from '../../../support/dictionary/testTypes';
+import devTeams from '../../../support/dictionary/devTeams';
+import NewExpenceClass from '../../../support/fragments/settings/finance/newExpenseClass';
+import SettingsFinance from '../../../support/fragments/settings/finance/settingsFinance';
+import FiscalYears from '../../../support/fragments/finance/fiscalYears/fiscalYears';
+import TopMenu from '../../../support/fragments/topMenu';
+import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
+import NewFiscalYear from '../../../support/fragments/finance/fiscalYears/newFiscalYear';
+import NewLedger from '../../../support/fragments/finance/ledgers/newLedger';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import InteractorsTools from '../../../support/utils/interactorsTools';
+
+describe('ui-finance: Add budget to fund', () => {
+  const firstExpenseClass = { ...NewExpenceClass.defaultUiBatchGroup };
+  const fund = { ...NewFund.defaultFund };
+  const defaultLedger = NewLedger.defaultLedger;
+  const defaultFiscalYear = { ...NewFiscalYear.defaultFiscalYear };
+
+  before(() => {
+    cy.loginAsAdmin();
+    cy.visit(SettingsMenu.expenseClassesPath);
+    SettingsFinance.createNewExpenseClass(firstExpenseClass);
+
+    cy.visit(TopMenu.fiscalYearPath);
+    FiscalYears.createDefaultFiscalYear(defaultFiscalYear);
```

Comment:
I don't see the removing of the fiscal year

Reply:
Done